include/fastdds/dds/publisher/qos/DataWriterQos.hpp

```diff
@@ -58,6 +58,12 @@ class RTPSReliableWriterQos
                (this->disable_heartbeat_piggyback == b.disable_heartbeat_piggyback);
     }
 
+    inline void clear()
+    {
+        RTPSReliableWriterQos reset = RTPSReliableWriterQos();
+        std::swap(*this, reset);
```

Comment:
I believe if you want to use `std::swap`, you should include `utility` to include what you use.